public/content/documentation/web/form/help-hint-error.md

```diff
@@ -0,0 +1,193 @@
+      - I HEAR when it appears dynamically, an error is read automatically
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/public/content/documentation/web/form/help-hint-error](https://www.magentaa11y.com/MagentaA11yV2#/public/content/documentation/web/form/help-hint-error)
+
+## Code examples
+
+### Adding hint/help text
+
+```html
+<label for="best-sesame-street-character">
+  The best Sesame Street character is:
+</label>
+<input type="text" 
+       id="best-sesame-street-character" 
+       aria-describedby="best-sesame-street-character-hint">
+
+<div class="hint" id="best-sesame-street-character-hint">
+  Example: Elmo, Big Bird, Cookie Monster, Grover, Oscar the Grouch
+</div>
+```
+
+<example>
+<label for="best-sesame-street-character">
+  The best Sesame Street character is:
+</label>
+<input type="text" 
+       id="best-sesame-street-character" 
+       aria-describedby="best-sesame-street-character-hint">
+
+<div class="hint" id="best-sesame-street-character-hint">
+  Example: Elmo, Big Bird, Cookie Monster, Grover, Oscar the Grouch
+</div>
+</example>
+
+### Adding an error
+
+**Note:** The alert must be structured as below to function properly in VoiceOver, with the alert text nested inside the `role="alert"` element.
+
+```html
+<label for="favorite-sesame-street-character-letter">
+  What is your Sesame Street character?
+  <span>Required</span>
+</label>
+
+<input type="text"
+       id="favorite-sesame-street-character"
+       aria-describedby="favorite-sesame-street-character-error favorite-sesame-street-character-hint"
+       required>
+
+<div role="alert" 
+     id="favorite-sesame-street-character-alert" 
+     class="alert inert">
+  <!--- Do not reference this alert element
+        directly with aria-describedby -->
+  <div id="favorite-sesame-street-character-error">
+    <!--- Use JS to inject the alert here -->
+  </div>     
+</div>
+
+<div class="hint" id="favorite-sesame-street-character-hint">
+  Example: Elmo, Big Bird, Cookie Monster, Grover, Oscar the Grouch
+</div>
+
+<button id="show-error">
+  Toggle error
+</button>
+```
+
+<example>
+<label for="favorite-sesame-street-character-letter">
+  What is your Sesame Street character?
+  <span>Required</span>
+</label>
+
+<input type="text"
+       id="favorite-sesame-street-character"
+       aria-describedby="favorite-sesame-street-character-error favorite-sesame-street-character-hint"
+       required>
+
+<div role="alert" 
+     id="favorite-sesame-street-character-alert" 
+     class="alert inert">
+  <!--- Do not reference this alert element
+        directly with aria-describedby -->
+  <div id="favorite-sesame-street-character-error">
+    <!--- Use JS to inject the alert here -->
+  </div>     
+</div>
+
+<div class="hint" id="favorite-sesame-street-character-hint">
+  Example: Elmo, Big Bird, Cookie Monster, Grover, Oscar the Grouch
+</div>
+
```

Comment:
Consider adding a TODO comment somewhere in here so we come back to the right spot.